BUILD_WSL.md

```diff
@@ -0,0 +1,31 @@
+# Compiling hashcat for Windows with Windows Subsystem for Linux.
+
+Tested on a Windows 10 (x64 build 18363), should also work to build hashcat for Windows on Linux.
+
+### Installation ###
+
+Enable WSL.
+
+Run bash (win+r bash) and make sure to install additional dependencies necessary for hashcat compilation
```

Comment:
press the win + r key on your keyboard simultaneously and in the "Run" popup window type bash